.gitignore

```diff
@@ -0,0 +1,3 @@
+pillar/config.sls
```

Comment:
Please base your PR on current master rather than other PRs unless necessary, this makes it easier for reviewers to accept individually.